core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt

```diff
@@ -17,13 +17,17 @@ class IFramePlayerOptions private constructor(private val playerOptions: JSONObj
         return playerOptions.toString()
     }
 
+    fun getOrigin(): String {
+        return playerOptions.getString(Builder.ORIGIN)
+    }
+
     class Builder {
         companion object {
             private const val AUTO_PLAY = "autoplay"
             private const val CONTROLS = "controls"
             private const val ENABLE_JS_API = "enablejsapi"
             private const val FS = "fs"
-            private const val ORIGIN = "origin"
+            const val ORIGIN = "origin"
```

Comment:
Why is this not private?

Reply:
because in other case I couldn't access ORIGIN constant from _IFramePlayerOptions_ to get origin value from _playerOptions_